cmd/ak/cmd/connections/delete.go

```diff
@@ -20,23 +18,16 @@ var deleteCmd = common.StandardCommand(&cobra.Command{
 		r := resolver.Resolver{Client: common.Client()}
 		c, id, err := r.ConnectionNameOrID(args[0])
 		if err != nil {
-			if errors.As(err, resolver.NotFoundErrorType) {
-				err = common.NewExitCodeError(common.NotFoundExitCode, err)
-			}
-			return err
+			return common.ToExitCodeError(err, "connection")
 		}
 		if !c.IsValid() {
-			err = errors.New("connection not found")
-			return common.NewExitCodeError(common.NotFoundExitCode, err)
+			return common.ToExitCodeError(sdkerrors.ErrNotFound, "connection")
```

Comment:
if you fix this line, maybe we should fix the error?

c.IsValid == not found? or invalid id ?